applications/nrf5340_audio/src/bluetooth/le_audio_cis_headset.c

```diff
@@ -694,9 +697,11 @@ int le_audio_send(struct encoded_audio enc_audio)
 		net_buf_unref(buf);
 		atomic_dec(&iso_tx_pool_alloc);
 	}
```

Comment:
Should return ret here and return 0 after the if, since that is what we usually does. helps with readability as well